### PR TITLE
Enable a trivial building test on TravisCI

### PR DESCRIPTION
**Pull request by mrakitin**
## Summary

This PR enables a trivial `conda build <dir>` test applied to the specific recipes updated within a PR (preferably we should always submit updates to a single recipe per PR). Even though we do a comprehensive cross-platform building/testing on Azure Pipelines using feedstocks, it's still worth to perform this light check on every PR.

## Motivation
After we mirror the changes from the corresponding feedstocks to be built on our internal machine `freyja`, we may still not get the built package in our old `lightsource2-tag` channel. Possible reasons:
- a mistake/typo in the `meta.yaml` file;
- a missing/wrong dependencies set;
- our internal building machine is down/unresponsive.

The first 2 points will be caught by the CI, the third point will be a clear reason of a missed build event on our internal server.

## Testing

This was verified in a separate branch https://github.com/mrakitin/lightsource2-recipes/tree/test-diffs. The test results are available at https://travis-ci.org/mrakitin/lightsource2-recipes/builds/586820615.

### CODE REVIEW

**Comment by mrakitin** [comment]
I'm requesting at least one review, please. Just wanted to bring it to everyone's attention. Would be great to merge it by the afternoon.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    git remote -v
+    git fetch upstream master
+    git status
+    git branch -avv
+    git log -n 3
+
+  - |
+    set -e
+
+    changed_files=$(git diff --name-only upstream/master -- | grep recipes-tag | grep meta.yaml || true)
```
Comment (danielballan) [inline .travis.yml]: This is clever.

Comment (mrakitin) [reply .travis.yml]: Thanks 🙂

**Comment by mrakitin** [comment]
Thanks for your review, @danielballan! Will wait for one more review before it gets merged.

**Comment by mrakitin** [comment]
Thanks @jklynch!

**Pull request merged**